courses/management/commands/manage_topics.py

```diff
@@ -0,0 +1,278 @@
+            course_title = row.get(COURSE_TITLE_COLUMN_NAME)
+            parent_topic1 = row.get(PARENT_TOPIC_1_COLUMN_NAME)
+            sub_topic1 = row.get(SUB_TOPIC_1_COLUMN_NAME)
+            parent_topic2 = row.get(PARENT_TOPIC_2_COLUMN_NAME)
+            sub_topic2 = row.get(SUB_TOPIC_2_COLUMN_NAME)
+            course_page = None
+            course_page_cls = ExternalCoursePage
+            if platform_name.lower() == DEFAULT_PLATFORM_NAME.lower():
+                course_page_cls = CoursePage
+            # Course titles can have trailing spaces so we would regex filter them instead of iexact
+            course_pages = course_page_cls.objects.filter(
+                title__regex=rf"(?i){course_title}\s*$",
+                course__platform__name=platform_name,
+            )
+
+            if course_pages:
+                parent_topic1 = get_topic(name=parent_topic1)
+                sub_topic1 = (
+                    get_topic(name=sub_topic1, parent=parent_topic1.name)
+                    if parent_topic1
+                    else None
+                )
+                parent_topic2 = get_topic(name=parent_topic2)
+                sub_topic2 = (
+                    get_topic(name=sub_topic2, parent=parent_topic2.name)
+                    if parent_topic2
+                    else None
+                )
+
+                for course_page in course_pages:
+                    assigned_topics_stats = ""
+                    skipped_topics_stats = ""
+                    if parent_topic1:
+                        course_page.topics.add(parent_topic1)
+                        assigned_topics_stats = (
+                            assigned_topics_stats + parent_topic1.name
+                        )
+                    if sub_topic1:
+                        course_page.topics.add(sub_topic1)
+                        assigned_topics_stats = (
+                            assigned_topics_stats + ", " + sub_topic1.name
+                        )
+
+                    # If sub_topic 2 is blank we only assign High Level topic 1 and Subtopic 1 (See: https://github.com/mitodl/hq/issues/5841#issuecomment-2447413927)
+                    if sub_topic2 and parent_topic2:
+                        course_page.topics.add(parent_topic2)
+                        course_page.topics.add(sub_topic2)
+                        assigned_topics_stats = (
+                            assigned_topics_stats
+                            + ", "
+                            + parent_topic2.name
+                            + ", "
+                            + sub_topic2.name
+                        )
+
+                    else:
+                        skipped_topics_stats = (
+                            skipped_topics_stats + parent_topic2.name
+                            if parent_topic2
+                            else None + "," + sub_topic2.name
+                            if sub_topic2
+                            else None
```

Comment:
Instead of appending these strings, We could have maintained a simple list and used the list to display the stats like `", ".join(assigned_topics_stats)`.